governance/polls/SparkLend Ethereum - Increase ETH borrow cap gap parameter to 20k - March 25, 2024.md

```diff
@@ -0,0 +1,55 @@
+---
+title: SparkLend Ethereum - Increase ETH borrow cap gap parameter to 20k - March 25, 2024
+summary: Signal your support or opposition to increasing the ETH borrow cap gap parameter to 20k ETH
```

Comment:
:green_circle: **nit**: we should not abbreviate values using `k`. Specify `20 thousand` or `20,000`.

```suggestion
summary: Signal your support or opposition to increasing the ETH borrow cap gap parameter to 20,000 ETH.
```